src/test/resources/sanity-tests/opensearch-node-key.pem

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
Why do we need new certs?

Reply:
these aren't new certs..these are generated by `install_demo_configuration.sh` script and I re-used them by placing them here. We need this, for the tests to be able to connect to the cluster.

Reply:
Instead of copying the files can we load the certs from the path they are stored?

Reply:
As of now they are hard-coded in the install script and get generated when the install script is run. Only other place they are stored is in bwc-test subproject. I didn't load it from that path as it is a separate sub-project.